app/graphql/types/mutation_type.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 module Types
-  class MutationType < Types::BaseObject
-    field :create_freak, FreakType, null: false, resolver: Resolvers::FreakCreateResolver
+  class MutationType < BaseObject
+    field :freak_create, FreakType, null: false, resolver: Resolvers::FreakCreateResolver
+    field :photo_create, Models::Photo, null: false, resolver: Resolvers::PhotoCreateResolver
```

Comment:
Interesting that you added this. We can keep it, but we didn't need it yet.

Reply:
We need this field for photoCreate mutation, which is used by `spec/controllers/graphql/mutations/photo_create_spec.rb` test.